testing/tests/DevExpress.ui.widgets.editors/autocomplete.tests.js

```diff
@@ -498,6 +501,7 @@ QUnit.test("arrow_down/arrow_up/enter provide item navigation and selection", fu
         .keyDown(KEY_UP);
 
     $selectedItem = $list.find(FOCUSED_STATE_SELECTOR);
+    assert.equal(isRenderer(instance._list.option("focusedElement")), config().useJQueryRenderer, "focusedElement is correct");
```

Comment:
This flag was renamed to useJQuery. It seems that tests will fail with jQuery strategy